guide/additional-info/changes-in-v14.md

```diff
@@ -0,0 +1,280 @@
+
+### Invite
+
+`#channel` and `#inviter` are now getters and resolve structures from the cache.
+
+### MessageComponent
+
+MessageComponents have been renamed as well. They no longer have the `Message` prefix:
+
+```diff
+- const button = new MessageButton();
++ const button = new ButtonComponent();
+
+- const selectMenu = new MessageSelectMenu();
++ const selectMenu = new SelectMenuComponent();
+
+- const actionRow = new MessageActionRow();
++ const actionRow = new ActionRow();
+```
+
+### MessageEmbed
+
+- `MessageEmbed` has now been renamed to `Embed`
+
+- `#setFooter` now accepts a sole `FooterOptions` object. (add link to dapi site)
+
+- `#addField` and `#addFields` both accept an object or array of `APIEmbedField`(s) respectively. (add link to dapi site)
+
+### PartialTypes
+
+The `PartialTypes` string array has been removed, instead use the `Partials` enum.
+
+### Permissions
+
+Thread permissions `USE_PUBLIC_THREADS` and `USE_PRIVATE_THREADS` have been removed as they are now deprecated in the API. Instead use the newer `*Threads` permission flags.
+
+### PermissionOverwritesManager
+
+Overwrites are now keyed by the `PascalCase` permission key rather than the `SCREAMING_SNAKE_CASE` permission key.
+
+### REST Events
+
+The following discord.js events:
+- `invalidRequestWarning`
+- `apiRequest`
+- `apiResponse`
+- `rateLimit`
+
+Have been removed from the `Client` in discord.js. Instead you should access these events from `Client#rest`. In addition `apiRequest`, `apiResponse` and `rateLimit` events have been renamed:
+
+```diff
+- client.on('apiRequest', ...);
++ client.rest.on('request', ...);
+
+- client.on('apiResponse', ...);
++ client.rest.on('response', ...);
+
+- client.on('rateLimit', ...);
++ client.rest.on('rateLimited', ...);
+```
+
+### RoleManager
+
+`Role.comparePositions` has been removed. Use `RoleManager#comparePositions` instead.
+
+### ThreadMemberManager
+
+`#fetch` now only takes a single object of type `ThreadMemberFetchOptions`.
+
+### Util
+
+`#removeMentions` has been removed, to control mentions you should use `allowedMentions` on `MessageOptions` instead.
+
+
+### `.deleted` Field(s) have been removed
+
+You can no longer use `#deleted` to check if a structure was deleted or not. 
+
+Check out [the issue ticket](https://github.com/discordjs/discord.js/issues/7091) for more context.
+
+### VoiceChannel
+
+`#editable` has been removed, instead you should use `GuildChannel#manageable` for checking this permission.
+
+Many of the analogous enums can be found be found in the discord-api-types docs (link website here)
+
+### VoicesRegion Changes
+
+`#vip` has been removed as the field is no longer part of the API.
+
+### Webhook
+
+`#fetchMessage` now only takes on sole object of type `WebhookFetchMessageOptions`.
+
+## Features
+
+### Channel
+
+New typeguards have been added:
+- `#isText`*
+- `#isDM`*
+- `#isGroupDM`
+- `#isVoice`*
+- `#isVoiceBased`
+- `#isTextBased`
```

Comment:
```suggestion
- `#isTextBased`
- `#isDMBased`
```

See https://github.com/discordjs/discord.js/pull/7362 (`.isDMBased()` has been added)

PS. I think this should be in alphabetical order?